x/wasm-storage/types/params.go

```diff
@@ -1,32 +1,36 @@
 package types
 
 import (
-	fmt "fmt"
+	"fmt"
+
+	"cosmossdk.io/errors"
 )
 
-const DefaultMaxWasmSize uint64 = 800 * 1024
+const (
+	DefaultMaxWasmSize uint64 = 800 * 1024
+	DefaultWasmTTL            = 259200 // 21 days
+)
 
 // DefaultParams returns default wasm-storage module parameters.
 func DefaultParams() Params {
 	return Params{
 		MaxWasmSize: DefaultMaxWasmSize,
+		WasmTTL:     DefaultWasmTTL,
 	}
 }
 
 // ValidateBasic performs basic validation on wasm-storage
 // module parameters.
-func (p Params) ValidateBasic() error {
-	err := validateMaxWasmSize(p.MaxWasmSize)
-	return err
+func (p *Params) ValidateBasic() error {
+	if p.WasmTTL < 2 {
+		return errors.Wrapf(ErrInvalidParam, "WasmTTL %d < 2", p.WasmTTL)
+	}
```

Comment:
why 2?

Reply:
2 is the least number of blocks that make sense. Say current block is N, and you add the wasm this block.

The time you know if it was sucessful or not, it will be block N+1.